schemas/org.gnome.shell.extensions.gtile.gschema.xml

```diff
@@ -38,133 +38,173 @@
+		      <default><![CDATA[['<Super><Control>KP_9']]]></default>
 		      <summary>Preset resize 19.</summary>
 		</key>
 		<key type="as" name="preset-resize-20">
 		      <default><![CDATA[['']]]></default>
 		      <summary>Preset resize 20.</summary>
 		</key>
+		<key type="as" name="preset-resize-21">
+		      <default><![CDATA[['<Super><Shift>KP_1']]]></default>
+		      <summary>Preset resize 21.</summary>
+		</key>
+		<key type="as" name="preset-resize-22">
+		      <default><![CDATA[['<Super><Shift>KP_2']]]></default>
+		      <summary>Preset resize 22.</summary>
+		</key>
+		<key type="as" name="preset-resize-23">
+		      <default><![CDATA[['<Super><Shift>KP_3']]]></default>
+		      <summary>Preset resize 23.</summary>
+		</key>
+		<key type="as" name="preset-resize-24">
+		      <default><![CDATA[['<Super><Shift>KP_4']]]></default>
+		      <summary>Preset resize 24.</summary>
+		</key>
+		<key type="as" name="preset-resize-25">
+		      <default><![CDATA[['<Super><Shift>KP_5']]]></default>
+		      <summary>Preset resize 25.</summary>
+		</key>
+		<key type="as" name="preset-resize-26">
+		      <default><![CDATA[['<Super><Shift>KP_6']]]></default>
+		      <summary>Preset resize 26.</summary>
+		</key>
+		<key type="as" name="preset-resize-27">
+		      <default><![CDATA[['<Super><Shift>KP_7']]]></default>
+		      <summary>Preset resize 27.</summary>
+		</key>
+		<key type="as" name="preset-resize-28">
+		      <default><![CDATA[['<Super><Shift>KP_8']]]></default>
+		      <summary>Preset resize 28.</summary>
+		</key>
+		<key type="as" name="preset-resize-29">
+		      <default><![CDATA[['<Super><Shift>KP_9']]]></default>
+		      <summary>Preset resize 29.</summary>
+		</key>
+		<key type="as" name="preset-resize-30">
+		      <default><![CDATA[['']]]></default>
+		      <summary>Preset resize 30.</summary>
+		</key>
 		
 		<key type = "s" name="resize1">
-		       <default>'2x2 0:0 0:0'</default>
+		       <default>'2x2 0:1 0:1'</default>
   	  	       <summary>Preset resize1 settings (example 6x4 1:2 3:3).</summary>
 		</key>       
 		<key type = "s" name="resize2">
-		       <default>'2x2 0:1 0:1'</default>
+		       <default>'2x2 0:1 1:1'</default>
   	  	       <summary>Preset resize2 settings.</summary>
 		</key>       
 		<key type = "s" name="resize3">
-		       <default>'2x2 1:0 1:0'</default>
+		       <default>'2x2 1:1 1:1'</default>
   	  	       <summary>Preset resize3 settings.</summary>
 		</key>       
 		<key type = "s" name="resize4">
-		       <default>'2x2 1:1 1:1'</default>
+		       <default>'4x4 0:0 0:3'</default>
   	  	       <summary>Preset resize4 settings.</summary>
 		</key>       
 		<key type = "s" name="resize5">
-		       <default>'3x3 0:0 0:2'</default>
+		       <default>'4x4 1:1 2:2'</default>
   	  	       <summary>Preset resize5 settings.</summary>
 		</key>   
 		<key type = "s" name="resize6">
-		       <default>'3x3 1:0 1:2'</default>
+		       <default>'4x4 3:0 3:3'</default>
```

Comment:
Same as above: wouldn't make more sense this to be ```2x2 1:0 1:1```?

Reply:
Yes, Super+Alt 4,5,6 should be set to something useful. But, not to duplicate Gnome built-in.